lib/suspenders.rb

```diff
@@ -1,5 +1,6 @@
 require 'suspenders/version'
 require 'suspenders/generators/app_generator'
+require 'suspenders/generators/static_generator'
```

Comment:
The generators need to be required in `lib/suspenders.rb` so that users can add `gem "suspenders"` to their `Gemfile` and this will give them the generators.

The _methods_ defined in `suspenders/generators/static_generator` are not called from within `suspenders/generators/app_generator.rb`, so we do not need to require it in there.